src/js/internal/primordials.js

```diff
@@ -129,6 +129,14 @@ export default {
       }
     },
   ),
+  SafeWeakMap: makeSafe(
```

Comment:
What is using this?

Reply:
[Calltracker](https://github.com/oven-sh/bun/blob/1ccc13ecf79861eb0b5a697c0b5571772d42a28d/src/js/internal/assert/calltracker.ts#L61)